.github/workflows/linux.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     - name: build
       run: |
         cd build
-        make -j 2
+        make VERBOSE=1 -j 2
```

Comment:
Are you planning to remove these again now that everything works? I don't feel strongly about it, but it produces quite a bit of output.

Reply:
I removed it again. I don't feel strongly either. It's easy enough to add it for debugging and it's not the point of this pull request.